app/components/administrative-unit-status.js

```diff
@@ -0,0 +1,13 @@
+import Component from '@glimmer/component';
+
+export default class AdministrativeUnitStatusComponent extends Component {
+  get getStatusSkin() {
+    if (this.args.status === 'Actief') {
```

Comment:
I would use the URI of the record instead of the label. I can imagine that the labels will change in the future while the URI _should_ always be the same, right?

An object which maps the URI to a skin value might be nice way to structure this:

```js
const ORGANIZATION_STATUS_SKINS = {
  'http://lblod.data.gift/concepts/abf4fee82019f88cf122f986830621ab': 'warning',
  // other types
}

export default class AdministrativeUnitStatusComponent extends Component {...}
```

And then you can simply return the correct skin by looking it up with the URI.

`return ORGANIZATION_STATUS_COLORS[someUriVariable];`